cmd/minishift/cmd/config/config.go

```diff
@@ -48,18 +48,19 @@ var settingsList []Setting
 
 var (
 	// minishift
-	ISOUrl           = createConfigSetting("iso-url", SetString, []setFn{validations.IsValidUrl}, []setFn{RequiresRestartMsg}, true)
-	CPUs             = createConfigSetting("cpus", SetInt, []setFn{validations.IsPositive}, []setFn{RequiresRestartMsg}, true)
-	Memory           = createConfigSetting("memory", SetString, []setFn{validations.IsValidMemorySize}, []setFn{RequiresRestartMsg}, true)
-	DiskSize         = createConfigSetting("disk-size", SetString, []setFn{validations.IsValidDiskSize}, []setFn{RequiresRestartMsg}, true)
-	VmDriver         = createConfigSetting("vm-driver", SetString, []setFn{validations.IsValidDriver}, []setFn{RequiresRestartMsg}, true)
-	OpenshiftVersion = createConfigSetting("openshift-version", SetString, nil, nil, true)
-	HostOnlyCIDR     = createConfigSetting("host-only-cidr", SetString, []setFn{validations.IsValidCIDR}, nil, true)
-	DockerEnv        = createConfigSetting("docker-env", SetSlice, nil, nil, true)
-	DockerEngineOpt  = createConfigSetting("docker-opt", SetSlice, nil, nil, true)
-	InsecureRegistry = createConfigSetting("insecure-registry", SetSlice, nil, nil, true)
-	RegistryMirror   = createConfigSetting("registry-mirror", SetSlice, nil, nil, true)
-	AddonEnv         = createConfigSetting("addon-env", SetSlice, nil, nil, true)
+	ISOUrl              = createConfigSetting("iso-url", SetString, []setFn{validations.IsValidUrl}, []setFn{RequiresRestartMsg}, true)
+	CPUs                = createConfigSetting("cpus", SetInt, []setFn{validations.IsPositive}, []setFn{RequiresRestartMsg}, true)
+	Memory              = createConfigSetting("memory", SetString, []setFn{validations.IsValidMemorySize}, []setFn{RequiresRestartMsg}, true)
+	DiskSize            = createConfigSetting("disk-size", SetString, []setFn{validations.IsValidDiskSize}, []setFn{RequiresRestartMsg}, true)
+	VmDriver            = createConfigSetting("vm-driver", SetString, []setFn{validations.IsValidDriver}, []setFn{RequiresRestartMsg}, true)
+	OpenshiftVersion    = createConfigSetting("openshift-version", SetString, nil, nil, true)
+	HostOnlyCIDR        = createConfigSetting("host-only-cidr", SetString, []setFn{validations.IsValidCIDR}, nil, true)
+	DockerEnv           = createConfigSetting("docker-env", SetSlice, nil, nil, true)
+	DockerEngineOpt     = createConfigSetting("docker-opt", SetSlice, nil, nil, true)
+	InsecureRegistry    = createConfigSetting("insecure-registry", SetSlice, nil, nil, true)
+	RegistryMirror      = createConfigSetting("registry-mirror", SetSlice, nil, nil, true)
+	AddonEnv            = createConfigSetting("addon-env", SetSlice, nil, nil, true)
+	HypervVirtualSwitch = createConfigSetting("hyperv-virtual-switch", SetString, nil, nil, true)
```

Comment:
Added hyperV virtual switch name to configuration and also as part of the start command. We do need now to modify the check around hyperV switch check where we are checking the `HYPERV_VIRTUAL_SWITCH` env variable.

Reply:
@praveenkumar , is it part of this PR or should be separate? if separate need to create issue then.